README.md

```diff
@@ -10,122 +10,108 @@ Android Auto for IDrive
 [![Buy Me A Coffee](https://img.shields.io/badge/support-buymeacoffee-5f7fff)](https://www.buymeacoffee.com/q4JVoxz)
 ![MIT Licensed](https://img.shields.io/github/license/hufman/AndroidAutoIdrive)
 
+What is it ?
+------------
+
+An extra integration point for your phone apps to better integrate with your car.
+
+How does it work ?
+------------------
+
 The BMW/Mini IDrive NBT does not offer native Android Auto integration, but does provide a very powerful Connected Apps convergence option with many exciting integration points into the car. This project is an effort to implement most of the features of Android Auto as unofficial BMW/Mini Connected Apps.
 
-By relying on the Connected Apps technology, this app greatly extends the functionality of the car without needing any modifications. Any MY2014 or newer BMW or Mini equipped with NBT or NBT Evo and the "Navigation System Professional (S609A)" [option](https://www.mdecoder.com/) or "BMW Apps (6NR)" or "Mini Connected (SA6NM)" options should be compatible.
+By relying on the Connected Apps technology, this app greatly extends the functionality of the car **without needing any modifications**.
```

Comment:
When I talked about the app on forums people were very suspicious because they thought it was some kind of IDrive software modification, that's why I felt the need to emphasize that it doesn't require anything but an app on your phone 😃

Indeed the wordy options list isn't very friendly, I think how you put it:

> "If your car adds the Connected and Calendar apps from your phone, it's ready to go!"

Sums it up pretty nicely!